doc/source/dev/contributor/development_workflow.rst

```diff
@@ -114,10 +114,10 @@ development environment, you'll need to activate your development virtual
 environment, perform an in-place build, and run tests::
 
    conda activate name-of-your-virtual-environment
-   python setup.py build_ext --inplace
-   python runtests.py -v
+   python dev.py build
+   python dev.py test -v
```

Comment:
Two separate commmands needed? A single `dev.py test` is strictly speaking enough